examples/miniapps/decoupled-packages/example/analytics/services.py

```diff
@@ -1,8 +1,18 @@
 """Analytics services module."""
 
 
-class AggregationService:
+from ..abstraction.analytics.services import AggregationServiceMeta
+from ..abstraction.photo.repositories import PhotoRepositoryMeta
+from ..abstraction.user.repositories import UserRepositoryMeta
 
-    def __init__(self, user_repository, photo_repository):
-        self.user_repository = user_repository
-        self.photo_repository = photo_repository
+
+class AggregationService(AggregationServiceMeta):
```

Comment:
```suggestion
class AggregationServiceImpl(AggregationService):
```
or
```suggestion
class MemoryAggregationService(AggregationService):
```
See comment above. This way it would be much more clear about implementation specifics of the `AggregationService` interface.